frontend/src/app/app.constants.ts

```diff
@@ -129,3 +129,13 @@ export const languages: Language[] = [
    { code: 'vi', name: 'Tiếng Việt' },      // Vietnamese
    { code: 'zh', name: '中文' },            // Chinese
 ];
+
+
+export const specialBlocks = {
+  '708998': {
```

Comment:
If I'm not mistaken.. 709632 is the actual block height of taproot activation.

Reply:
Ops, force pushed with my test block! Will change to 709632!